frontend/components/TableContainer/DataTable/SetupScriptProcessCell/SetupScriptProcessCell.tsx

```diff
@@ -0,0 +1,21 @@
+import Graphic from "components/Graphic/Graphic";
+import React from "react";
+
+const baseClass = "setup-script-process-cell";
+
+interface ISetupScriptProcessCell {
+  name: string;
+}
+
+const SetupScriptProcessCell = ({ name }: ISetupScriptProcessCell) => {
+  return (
+    <span className={baseClass}>
+      <Graphic name="file-sh" className={`${baseClass}__icon`} />
```

Comment:
I grabbed this from the Setup Experience admin UI; we don't have an `<Icon>` for it.